### DISCUSSION TITLE
Can't set resources direction in preference

### DISCUSSION THREAD

**Discussion by cswwp** [ZuodaoTech/everyone-can-use-english#824] [Q&A]
Hi, @an-lee  i had install enjoy app with version 0.2.13 , but i can't set resources direction in preference, any tips? thank you.
<img width="1022" alt="image" src="https://github.com/ZuodaoTech/everyone-can-use-english/assets/11901683/0c9929f1-2b5d-439b-9262-96c3abcf59ca">

**Comment by an-lee** [comment]
You mean the `Libary Path`?  It has been moved the `Account` section.
